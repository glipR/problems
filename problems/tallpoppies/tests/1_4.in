6 41 50
151113766 353502621 79608090 48869907 358679260 31197234 760286402 505081617 729273987 638016089 715604921 323607613 604322747 845188920 819648396 434451942 953487343 444546201 126490393 200937281 533864347 115064261 425121421 326883035 775742128 178147721 927375369 195212308 907418649 834423926 797538722 14879734 350809147 954811762 254580170 342467421 575483430 15991489 50745827 158196631 81609027
1885948 1 613193951
10158385 1 199073368
10726524 0
16686325 1 83094493
44637944 0
60694049 1 320343117
87992654 1 861370143
103470350 1 372247322
104452340 1 328405980
176508793 1 464054319
207104964 1 521351881
210865044 1 629388689
211461778 0
255888791 0
257553995 1 719752886
266311940 0
269337655 1 710494886
339943195 1 857174693
364488409 0
416651380 1 99787486
435126996 1 435198415
441517712 1 872127169
456497778 0
484430180 0
528320197 0
536808937 0
551012252 1 918464509
571256942 1 273035615
574515869 1 694145694
586131209 0
677121263 0
678917011 0
699235175 0
770529683 0
866782637 0
873071335 1 378894268
877022695 0
877834549 1 260403648
880271268 0
880414461 0
904492008 0
926517818 1 163943242
942422225 1 764203614
942517050 0
949403075 1 609250817
965978619 0
972165786 1 735229487
975415158 0
982981220 1 982080482
999552911 0
