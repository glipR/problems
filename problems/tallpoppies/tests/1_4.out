1019170906
1221315007
2196146037
2442751395
2443060560
3021354380
3558635064
3704119802
3945663310
3981140024
4246061241
4778332499
4711518155
4799733797
5225056361
5776179252
5796000517
5786709225
5727032967
5861403990
6080223721
6154550974
6205993569
6348126613
